whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceEndToEndTest.java

```diff
@@ -678,27 +677,6 @@ public void apply(String input, Deque<String> result) {
                 "<credential>PasswordCredential</credential>"));
     }
 
-    @Test
-    public void create_inetnum_with_parent_without_status_to_check_error_message_beginning_with_percent_is_handled_correctly() {
```

Comment:
Should this test be removed? It's testing the error message, not the missing status

Reply:
It was creating a situation that cannot be created and then checking against (parent has no status). All that logic has been removed as it is no longer possible to create these types of situations and none exist in the data anymore. Test is therefore obsolete.